crates/core/src/mock/mock_starknet.rs

```diff
@@ -38,7 +42,9 @@ impl From<AvailableFixtures> for JsonRpcMethod {
     fn from(value: AvailableFixtures) -> Self {
         match value {
             AvailableFixtures::Other(method) => method,
-            AvailableFixtures::ComputeStarknetAddress | AvailableFixtures::GetEvmAddress => JsonRpcMethod::Call,
+            AvailableFixtures::ComputeStarknetAddress
```

Comment:
Isn't `get_nonce` also a call?
Maybe the contract_account.nonce() method deserves to be unitarily tested

Reply:
Gotcha will add a unit test